src/main/webapp/app/shared/metis/postings-markdown-editor/postings-markdown-editor.component.html

```diff
@@ -12,5 +12,7 @@
     ></jhi-markdown-editor>
 </div>
 <div class="col text-end">
-    <p class="small my-0 ms-2">{{ content?.length }} / {{ maxContentLength }}</p>
+    <p class="small my-0 ms-2" [ngStyle]="{ color: content?.length! > maxContentLength || content?.length! == 0 ? 'red' : 'black' }">
+        {{ content?.length }} / {{ maxContentLength }}
```

Comment:
If content is undefined we actually display undefined, which is probably not expected behaviour, right?

Reply:
Ah yes if that what it looks like i think it's ok, I initially thought there would be a printed undefined :)